i/imagemagick/manifest.x86_64.jsonc

```diff
@@ -51,6 +51,9 @@
 				"/usr/bin/mogrify",
 				"/usr/bin/montage",
 				"/usr/bin/stream",
+				"/usr/lib/libMagick++-7.Q16HDRI.la",
```

Comment:
We only need the la files in the modules directory, not the ones in libdir. Can you please add a `rm -v %(installroot)%(libdir)/*.la` to the install section?

Reply:
And anyway even if we didn't delete them here we would have wanted them patterned into the same package the libs are in.